KerbalStuff/objects.py

```diff
@@ -121,9 +121,6 @@ def is_active(self) -> bool:
     def is_anonymous(self) -> bool:
         return False
 
-    def get_id(self) -> str:
-        return self.username
```

Comment:
http://localhost:5080/login
```
AttributeError

AttributeError: 'User' object has no attribute 'get_id'
Traceback (most recent call last)

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2091, in __call__

    return self.wsgi_app(environ, start_response)

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2076, in wsgi_app

    response = self.handle_exception(e)

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app

    response = self.full_dispatch_request()

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request

    rv = self.handle_user_exception(e)

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1393, in handle_user_exception

    return self.ensure_sync(handler)(e)

    File "/opt/spacedock/KerbalStuff/app.py", line 171, in handle_generic_exception

    raise e

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request

    rv = self.dispatch_request()

    File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request

    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)

    File "/opt/spacedock/KerbalStuff/blueprints/accounts.py", line 142, in login

    login_user(user, remember=remember)

    File "/usr/local/lib/python3.8/site-packages/flask_login/utils.py", line 169, in login_user

    user_id = getattr(user, current_app.login_manager.id_attribute)()

AttributeError: 'User' object has no attribute 'get_id'
```

Reply:
Ahh, that's why that was there. It caused errors with PyType and I couldn't find anyplace we were using it, so I took it out when I was still trying to get that to work. I'll put it back.

Reply:
Done.